libparsec/crates/platform_http_proxy/Cargo.toml

```diff
@@ -14,4 +14,8 @@ workspace = true
 [dependencies]
 libparsec_types = { workspace = true }
 
-reqwest = { workspace = true }
+reqwest = { workspace = true, features = ["default-tls"] }
```

Comment:
why this feature here ? it should be up to `libparsec_client_connection` to choose which tls flavor to rely on

Reply:
Done